src/main/java/orca/flukes/GUIUnifiedState.java

```diff
@@ -948,6 +948,7 @@ public void run() {
 			} else if (e.getActionCommand().equals("color")) {
 				linkCreator.setLinkType(OrcaLinkCreator.OrcaLinkType.COLOR);
 			} else if (e.getActionCommand().equals("autoip")) {
+				removeAssignedRequestIPs();
```

Comment:
How will this affect slices that have already been created / are currently being modified?

Reply:
This code only removes IPs assigned in the current stage, i.e. IP addresses that were assigned previously and already submitted for modification should not be affected.

Reply:
If an IP address is already in the manifest because it was assigned during a previous step, it is not affected.